### DISCUSSION TITLE
Configuring cobbler with api

### DISCUSSION THREAD

**Discussion by martux69** [uyuni-project/uyuni#5983] [Devel]
Hi,
I would like to configure the kernel option in the distribution by using an api call. Which URL must I use for this?
As an alternative I try to configure the kernel option b ythe cobbler cli, but the change is not reflected so far in the Uyuni web gui.
In particular I would like to set the install parameter in the kernel option to another path.
Any help is welcome :-)
Kind regards
Martin

**Comment by SchoolGuy** [comment]
So the change in kernel options should be reflected in the WebUI. Both Uyuni and Cobbler have API calls available that should change the kernel options in Cobbler. Overwriting the `install=...` parameter is not possible with Uyuni since that is overwritten by Uyuni.
